Dragonflight/WarriorArms.lua

```diff
@@ -767,7 +825,10 @@ spec:RegisterAbilities( {
             applyDebuff( "target", "deep_wounds" )
             if talent.in_for_the_kill.enabled then
```

Comment:
This needs to be changed to only apply the haste bonus if In for the Kill isn't already applied.  Otherwise, it'll distort recommendations that follow.

Reply:
CS fixed in https://github.com/Hekili/hekili/pull/1670/commits/33370bc31a64a614f1d30da0c18027f72e0beb2f
WB fixed in https://github.com/Hekili/hekili/pull/1670/commits/4cc68109cde878a259f4b46c5c1ad90194cca215